services/cd-service/pkg/repository/repository_test.go

```diff
@@ -282,6 +360,50 @@ func TestNew(t *testing.T) {
 	}
 }
 
+func TestBootstrapError(t *testing.T) {
+	tcs := []struct {
+		Name          string
+		ConfigContent string
+	}{
+		{
+			Name:          "Invalid json in bootstrap configuration",
+			ConfigContent: `{"development": "upstream": {"latest": true}}}`,
```

Comment:
maybe add another test with a json syntax error?

Reply:
how is json syntax one different from this one?

Reply:
yes, you are right.